packages/gatsby-cli/src/create-cli.ts

```diff
@@ -534,6 +508,95 @@ Creating a plugin:
         report.log(`Telemetry collection ${enabled ? `enabled` : `disabled`}`)
       }),
     })
+    .command({
```

Comment:
Not for reviewers that in the course of doing this I flagged this command as broken at the top level. So @wardpeet recommended I fix it along with this change.